csharp/src/Ice/LoggerAdmin.cs

```diff
@@ -47,42 +49,43 @@ internal bool IsAccepted(LogMessage logMessage)
         /// <param name="localLogger">The local logger.</param>
         /// <param name="request">A delegate that represents an asynchronous request to be called on the remote
         /// logger.</param>
-        internal void Queue(string operation, ILogger localLogger, Func<IRemoteLoggerPrx, ValueTask> request)
+        internal void Queue(string operation, ILogger localLogger, Func<IRemoteLoggerPrx, Task> request)
         {
             lock (_mutex)
             {
                 // Creates a new task that waits for the last request task to complete and to send the given
                 // request. Save this new task as the last request task.
-                _lastRequestTask = QueueAsync(_lastRequestTask, _loggerAdmin, _remoteLoggerPrx, operation, localLogger,
+                _lastRequestTask = QueueAsync(_lastRequestTask, operation, localLogger,
                      request);
             }
 
-            static async ValueTask QueueAsync(ValueTask previousRequestTask, LoggerAdmin loggerAdmin,
-                IRemoteLoggerPrx proxy, string operation, ILogger localLogger,
-                Func<IRemoteLoggerPrx, ValueTask> request)
+            async Task QueueAsync(Task previousRequestTask, string operation, ILogger localLogger,
```

Comment:
Sending async oneway request might indeed complete synchronous but since our mapping for XxxxAsync returns a Task so I switched to Task. Should we fix our mapping to use ValueTask?

Reply:
Ok, let's keep `Task` / `Task<T>` for the AMI mapping and for `QueueAsync`.